### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ArtMot99**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,10 +1,49 @@
...
 
 def main() -> None:
-    pass
+    # Open and load JSON file to "players" variable
+    with open("players.json") as players_file:
+        players = json.load(players_file)
+        for player in players:
+            race = players[player]["race"]
+            skills = players[player]["race"]["skills"]
+            guild = players[player]["guild"]
```
Comment (ArsenPidhoretskyi) [inline main.py:13]: Use .items() or .values() for dict. Getting in the loop by key the same dict like: `players[player]` is so bad

File: main.py [hunk 2]
```diff
@@ -1,10 +1,49 @@
...
+            # Create Race objects if records not exist yet
+            if not Race.objects.filter(name=race["name"]).exists():
+                Race.objects.create(
+                    name=race["name"],
+                    description=race["description"]
+                )
```
Comment (ArsenPidhoretskyi) [inline main.py:19]: You can use .get_or_create
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create
The same in other places

File: main.py [hunk 3]
```diff
@@ -1,10 +1,49 @@
...
+            # Create Guild objects if records not exist yet
+            if guild:
```
Comment (ArsenPidhoretskyi) [inline main.py:21]: Can you explain for what you check?

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: main.py (hunk 3 above)
Comment (ArtMot99) [reply main.py:21]: If the character in the json file has a guild equal to null, then at the stage of creating an entry in the table I will get an error, here I filter out null values.

File: main.py (hunk 2 above)
Comment (ArtMot99) [reply main.py:19]: Снимок экрана 2023-02-07 в 11 28 16

**Review by ArsenPidhoretskyi — APPROVED** [review]
Well done!

**Pull request closed without merging**